apps/pie-docs/src/assets/styles/components/_card.scss

```diff
@@ -44,6 +76,36 @@ h2 + .c-card-wrapper {
     }
 }
 
+.c-card-icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 48px;
+    width: 48px;
```

Comment:
you could probably just defined the height or width and then set `aspect-ratio: 1 / 1;` I think

Reply:
😮